site/content/docs/5.0/migration.md

```diff
@@ -181,7 +182,7 @@ Badges were overhauled to better differentiate themselves from buttons and to be
 
 ## Accessibility
 
-- `.sr-only-focusable` does not require `.sr-only` anymore. [See #28720](https://github.com/twbs/bootstrap/pull/28720).
+- `.v-hidden-focusable` does not require `.v-hidden` anymore. [See #28720](https://github.com/twbs/bootstrap/pull/28720).
```

Comment:
This is not 100% correct anymore, we might need to rename this to _`.v-hidden-focusable` can be used stand alone_ or something alike

Reply:
isn't it true from a developer/user point of view? as in "you don't need to use v.hidden anymore"? think the meaning is correct, but you're looking at this from a systems point of view?

Reply:
happy to reword though either way, yes

Reply:
`.v-hidden` just didn't exist in v4, so it's weird to say it's not required anymore. A little rewording 'll do the job

Reply:
oh i see what you mean now, yes

Reply:
done/clarified, I think